### DISCUSSION TITLE
Table Filter as a dropdown with textual values

### DISCUSSION THREAD

**Discussion by RaphaelVanierBentley** [iTwin/iTwinUI#1239] [Q&A]
Hello, 

I am currently the <Table ...> tag along with a text filter on one of the columns. I would like to transform that text filter into a dropdown filter with a set of values as I have only 3 possible values for this column. Example, I would like to have dropdown for either "Achieved", "Failed" and "Skipped".

Is this possible?

**Comment by jzbuchalski** [comment]
Something like:
![image](https://github.com/iTwin/iTwinUI/assets/17307464/27fe211b-7539-4acf-8268-1b8b325da357)

I do not have a sandbox setup, but you can take a look at:

https://bentleycs.visualstudio.com/beconnect/_git/TCDEAppService?path=packages/modules/itwin-forms/src/components/grid/filters/FormsStateFilter.tsx&version=GC4ac2e59d426665f926b0703c009360f6218c9801&_a=contents

Add something like this to your column:
      {
        id: xx,
        Filter: FormsStateFilter,
        filter: "includesValue",
      },
